quesoqueue.h

```diff
@@ -64,6 +64,6 @@ class QuesoQueue {
     bool isOnline(Level l);
     std::deque<Level> _levels;
     std::optional<Level> _current;
-    Twitch _twitch;  // query online state
+    Twitch* _twitch;  // query online state
```

Comment:
Why switch to pointer? Is there a reason we want to manually manage the lifetime now?

Reply:
I wanted to make sure that the same instance was being used by both quesoqueue and chat.  I'll happily admit that I still don't largely understand how these things work so if switching to a const reference is a better idea, then I'm happy to do it.  I just thought that pointers were how objects like this were shared.

Reply:
It's the same instance. const reference is a pointer that is more idiotproof. It's still getting a pointer under the covers, but this way you can treat it like an object (. notation) and you don't have the ability to accidentally free it early and screw up somewhere else it's being used.

Reply:
Done